scio-core/src/main/scala/com/spotify/scio/io/Taps.scala

```diff
@@ -70,9 +73,9 @@ trait Taps {
   private def tableExists(table: TableReference): Boolean =
     Try(BigQueryClient.defaultInstance().getTableSchema(table)).isSuccess
 
-  private def bigQueryTap(sqlQuery: String): BigQueryTap = {
+  private def bigQueryTap(sqlQuery: String, flattenResults: Boolean): BigQueryTap = {
```

Comment:
I wonder if it makes sense to expose JobConfigurationQuery instead - what if we have to expose more options?